docs/guides/sdk/authentication.md

```diff
@@ -0,0 +1,181 @@
+---
+contributors: Kevin Lewis, Tim de Heiden
+description: Learn about all of the ways to control authentication with the Directus SDK
+---
+
+# Directus SDK Authentication
+
+The Directus SDK provides multiple ways to [authenticate your requests](/reference/authentication), by either generating
+an access and refresh tokens through login, or by setting a static token.
+
+## Generate, Use, and Refresh Access Tokens
+
+### Using the `authentication()` Composable
+
+The `authentication()` composable provides the Directus client with new methods related to authentication. It also
+manages token storage and refreshing on your behalf.
+
+```js
+import { createDirectus, authentication } from '@directus/sdk';
+
+const client = createDirectus('http://directus.example.com').with(authentication());
+
+await client.login(email, password);
+await client.refresh();
+await client.logout();
+```
+
+This approach will handle refreshing of the token automatically. The current token is stored inside the initialized
+client.
+
+### Using the `rest()` Composable
+
+```js
+import { createDirectus, rest, login, refresh, logout } from '@directus/sdk';
+
+const client = createDirectus('http://directus.example.com').with(rest());
+
+const user = await client.request(login(email, password));
+await client.request(refresh(mode, refresh_token));
+await client.request(logout(mode, refresh_token));
```

Comment:
Thanks for the update ❤️ 
However, the `logout()` method only requires the `refresh_token`:
```suggestion
await client.request(logout(refresh_token));
```